sky/cli.py

```diff
@@ -1807,7 +1838,8 @@ def _terminate_or_stop_clusters(
     operation = 'Terminating' if terminate else 'Stopping'
     if idle_minutes_to_autostop is not None:
         verb = 'Scheduling' if idle_minutes_to_autostop >= 0 else 'Cancelling'
-        operation = f'{verb} auto-stop on'
+        down_str = ' (terminate)' if terminate else ''
+        operation = f'{verb} auto-stop{down_str} on'
```

Comment:
Wdyt about
```
> sky autostop --terminate
Scheduling auto-stop (terminate) on ..
```
vs.
```
> sky autostop --down / --rm
Scheduling auto-down on ..
```
? The last code block suggests `--down` may be better than `--rm`.

Reply:
I think `auto-down` can be a good option. Updated the log.